src/main/resources/migrations/00022_alter_table_invites_add_column_disabled.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter-table-invites-add-column-disabled
+ALTER TABLE invites ADD COLUMN disabled BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Yes, currently we don't have invites in production (or test/staging environments), that is why no migration needed.